pkg/apis/allocation/v1/gameserverallocation.go

```diff
@@ -77,6 +77,12 @@ type GameServerAllocationSpec struct {
 	// This is useful for things like smoke testing of new game servers.
 	Preferred []GameServerSelector `json:"preferred,omitempty"`
 
+	// Ordered list of GameServer label selectors.
```

Comment:
Can you mark Required and Preferred (above) as deprecated with a pointer to this new field.

Reply:
Done.